modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java

```diff
@@ -209,6 +216,19 @@ public CPDefinitionInventory updateCPDefinitionInventory(
 		return cpDefinitionInventoryPersistence.update(cpDefinitionInventory);
 	}
 
+	private void _validateOrderQuantity(
+			int minOrderQuantity, int maxOrderQuantity,
+			int multipleOrderQuantity)
+		throws CommerceOrderQuantityException {
+
+		if ((minOrderQuantity < 1) || (maxOrderQuantity < 1) ||
+			(multipleOrderQuantity < 1)) {
+
+			throw new CommerceOrderQuantityException(
+				"Order Quantity must be greater than  or equal to 1");
```

Comment:
I don't think you are giving the right message doing like this, you should have an if case for each quantity
```suggestion
		if ((minOrderQuantity < 1){

			throw new CommerceOrderQuantityException(
				"Minimum Order Quantity must be greater than  or equal to 1");
				}
		if( maxOrderQuantity < 1) {
					throw new CommerceOrderQuantityException(
				"Maximum Order Quantity must be greater than  or equal to 1");
				}
				if( multipleOrderQuantity < 1) {
					throw new CommerceOrderQuantityException(
				"Multiple Order Quantity must be greater than  or equal to 1");
				}
		

```

Reply:
@victorcdp , see requested change.

Reply:
@andrea-ale-sbarra , required changes have been made.